bluebottle/bluebottle_drf2/serializers.py

```diff
@@ -40,10 +40,10 @@ def to_internal_value(self, data):
 
 class SorlImageField(RestrictedImageField):
     def __init__(self, geometry_string, **kwargs):
-        self.crop = kwargs.pop('crop', 'center')
```

Comment:
The idea is that we could pass kwargs like here:
https://github.com/onepercentclub/bluebottle/blob/c87da0d0b79c55e3f79cb98bcfd9129c6be5bbef/bluebottle/members/serializers.py#L42